neo/Prompt/Commands/Wallet.py

```diff
@@ -279,6 +280,95 @@ def command_desc(self):
         return CommandDesc('alias', 'create an alias for an address', params=[p1, p2])
 
 
+class CommandWalletExport(CommandBase):
+
+    def __init__(self):
+        super().__init__()
+        self.register_sub_command(CommandWalletExportWIF())
+        self.register_sub_command(CommandWalletExportNEP2())
+
+    def command_desc(self):
+        return CommandDesc('export', 'export wallet items')
+
+    def execute(self, arguments):
+        item = get_arg(arguments)
+
+        if not item:
+            print(f"Please specify an action. See help for available actions")
+            return False
+
+        try:
+            return self.execute_sub_command(item, arguments[1:])
+        except KeyError:
+            print(f"{item} is an invalid parameter")
+            return False
+
+
+class CommandWalletExportWIF(CommandBase):
+    def __init__(self):
+        super().__init__()
+
+    def execute(self, arguments):
+        wallet = PromptData.Wallet
+
+        if len(arguments) != 1:
+            print("Please specify the required parameter")
+            return False
+
+        address = arguments[0]
+        keys = PromptData.Wallet.GetKeys()
```

Comment:
Could use `wallet` here or remove the definition of `wallet` local var which is unused.